packages/server/package.json

```diff
@@ -90,7 +90,7 @@
     "webpack-hot-middleware": "^2.22.2",
     "webpack-node-externals": "2.5.2",
     "webpack-s3-plugin": "^1.0.0-rc.0",
-    "wrkbx": "^4.3.1-1"
+    "workbox-webpack-plugin": "^6.5.3"
```

Comment:
Since your usage of `workbox` appears to just be in generating the service worker, there wasn't much else that needed to change to upgrade these two major versions. Here are the migration guides in case you want to review anyway:

[v4 -> v5](https://developer.chrome.com/docs/workbox/migration/migrate-from-v4/)
[v5 -> v6](https://developer.chrome.com/docs/workbox/migration/migrate-from-v5/)